src/api/interfaces/LoginUser.ts

```diff
@@ -0,0 +1,4 @@
+export interface LoginUser {
```

Comment:
I think using a suffix may be helpful to indicate whether it's a request type or a response type, ex. LoginUserReq.

Reply:
These interfaces are not only used in api hooks